mobile/src/main/java/org/openhab/habdroid/background/ItemUpdateWorker.kt

```diff
@@ -71,13 +78,9 @@ class ItemUpdateWorker(context: Context, params: WorkerParameters) : Worker(cont
         }
 
         val itemName = inputData.getString(INPUT_DATA_ITEM_NAME)!!
-        var value = inputData.getString(INPUT_DATA_VALUE)!!
 
-        var label = inputData.getString(INPUT_DATA_LABEL)
-        if (label.isNullOrEmpty()) label = itemName
-
-        var mappedValue = inputData.getString(INPUT_DATA_MAPPED_VALUE)
-        if (mappedValue.isNullOrEmpty()) mappedValue = value
+        var valueForLog: String? = null
+        var valueToBeSentForLog: String? = null
```

Comment:
I find those additional variables slightly cumbersome. How about logging `value.value` and `valueToBeSent` before the actual HTTP call? Doing so avoids the weird variables (at the expense of having two log lines, which IMHO is OK).